pypeit/par/pypeitpar.py

```diff
@@ -1610,19 +1611,38 @@ def __init__(self, obj_toler=None, assign_obj=None,
         # Fill out parameter specifications.  Only the values that are
         # *not* None (i.e., the ones that are defined) need to be set
 
-        defaults['obj_toler'] = 5.
+        defaults['obj_toler'] = 1.
         dtypes['obj_toler'] = float
         descr['obj_toler'] = 'Tolerance (arcsec) to match source to targeted object'
```

Comment:
Please make it more clear in the docs that this parameter  is only used when mask identification information is provided.